### ISSUE TITLE
Selecting a new isotope triggers post-decay context description with invalid time value (a11y view)

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/beta-decay#47]
**Test device**
Asus Zenbook

**Operating System**
Win 11

**Browser**
Firefox

**Problem description**
From https://github.com/phetsims/qa/issues/1435

When switching between isotope types after an isotope decays, the description repeats the previously decayed time value (in years/days )
 
**Steps to reproduce**
1. Go to the A11y view.
2. Go to the Single Atom screen.
3. Select the Hydrogen-3 isotope.
4. Click on Add atom. You will see this context description: "Electron and antineutrino emitted from nucleus. Decay time, 0.60 years."
5. Select the carbon-14 isotope. Once you click on it this ctx description is shown: "Electron and antineutrino emitted from nucleus. Decay time, 1 years." or sometimes you will get  "Electron and antineutrino emitted from nucleus. Decay time, 0 years."

**Visuals**

https://github.com/user-attachments/assets/b7bd671b-46b4-40c0-af63-f90656aeb57f

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Beta Decay‬
URL: https://phet-dev.colorado.edu/html/beta-decay/1.0.0-dev.7/phet/beta-decay_all_phet.html
Version: 1.0.0-dev.7 2026-08-28 14:31:20 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:154.0) Gecko/20100101 Firefox/154.0
Language: es-ES
Window: 1440x768
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (ANGLE (AMD, Radeon R9 200 Series Direct3D11 vs_5_0 ps_5_0), or similar)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>

**Comment by jbphet** [comment]
This was caused by having the context response hooked to a derived property that was getting updated when the isotope selection was changed but before the last decay time had been reset.  I've changed this to be triggered _only_ by a change to the `lastDecayTimeProperty` that indicates a decay and then to use the time configuration to format the time value.  Assigning to @AgustinVallejo for review.

**Comment by AgustinVallejo** [comment]
Looking good. Back to VP for double checking and closing!

**Comment by vperezcadavid99** [comment]
It looks good. Thanks!
Closing.
